data/json/mapgen/park.json

```diff
@@ -76,48 +97,363 @@
         "object": {
             "fill_ter": "t_pavement",
             "rows": [
-                "                        ",
-                "|-+------------------+-|",
-                "|     .  . 7 .  .      |",
-                "|     .  .   .  .      |",
-                "|#    .  .....  .     #|",
-                "|#    .         .     #|",
-                "|#    .         .     #|",
-                "|#    .         .     #|",
-                "|#    .         .     #|",
-                "|#     .       .      #|",
-                "|#      .     .       #|",
-                "|......................|",
-                "|#      .     .       #|",
-                "|#     .       .      #|",
-                "|#    .         .     #|",
-                "|#    .         .     #|",
-                "|#    .         .     #|",
-                "|#    .         .     #|",
-                "|#    .  .....  .     #|",
-                "|     .  .   .  .      |",
-                "|     .  . 7 .  .      |",
-                "|-+------------------+-|",
-                "                   V    ",
-                "                        "
+				"                        ",
+				" ---------------------- ",
+				" +  . .   . 7 .   . . + ",
+				" |V . .   .   .   . . | ",
+				" |  . .   .....   . . | ",
+				" |  .  .  .   .  .  . | ",
+				" |H .   .  ...  .   .H| ",
+				" |H .    .......    .H| ",
+				" |H .               .H| ",
+				" |H .      ...      .H| ",
+				" |H .     .   .     .H| ",
+				" |  ................. | ",
+				" |H .     .   .     .H| ",
+				" |H .      ...      .H| ",
+				" |H .               .H| ",
+				" |H .    .......    .H| ",
+				" |H .   .  ...  .   .H| ",
+				" |  .  .  .   .  .  . | ",
+				" |  . .   .....   . . | ",
+				" |R . .   .   .   . . | ",
+				" +  . .   . 7 .   . . + ",
+				" ---------------------- ",
+				"                        ",
+				"                        "
             ],
             "terrain": {
                 ".": "t_pavement_y",
                 "7": "t_backboard",
                 "|": "t_chainfence_v",
                 "-": "t_chainfence_h",
-                "+": "t_chaingate_l"
+                "+": "t_chaingate_c",
+				"H": "t_pavement",
```

Comment:
I beg your pardon, I'm completely new to this. Should I make and commit the suggested changes and do another pull request? Or is there another way I should go about resolving these?